ios/FluentUI/Bottom Sheet/ContentHeightResolutionContext.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContentHeightResolutionContext)
+public class ContentHeightResolutionContext: NSObject {
```

Comment:
add documentation on this public class please?

Reply:
Done